.circleci/config.yml

```diff
@@ -76,23 +113,55 @@ jobs:
       - run:
           name: Lint Client and Server
           command: |
-            ./node_modules/.bin/eslint-eslines $(git diff --name-only --diff-filter=d origin/master... | grep -E '^(client|server)' | grep -E '\.jsx?$' | circleci tests split)
+            # We'll use failure code to do some additional processing and report failure manually
+            set +o errexit
+
+            # Run the lint
+            npx --no-install                                          \
+              eslint-eslines                                          \
+              $(                                                      \
+                git diff --name-only --diff-filter=d origin/master... \
+                | grep -E '^(client|server)'                          \
+                | grep -E '\.jsx?$'                                   \
+                | circleci tests split                                \
+              )                                                       \
```

Comment:
fun fact, if we need to we can diff the output of two command and get the list of lines in one but not the other (if we need to)

```bash
man comm

comm -23 <(git diff --name-only | sort -u) <(git diff --name-only --cached | sort -u)
```